### ISSUE TITLE
achievements

### ISSUE THREAD

**Issue by officialskptv-dev** [blizzless/blizzless-diiis#56]
not sure about the achievements but everytime i press y it says replace with game data

**Comment by advocaite** [comment]
yeah the new achievement data doesn't actually download i think we can fix it by using older achivment version but its something we need to figure out

**Comment by iamdroppy** [comment]
I'll investigate.
